src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -55,8 +53,11 @@
 {/if}
 
 {#if actions}
-	<div transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}>
-		<Actions {send} />
+	<div
+		transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}
+		class="flex w-full justify-center"
```

Comment:
Adding the class preserves the smoothness of the animation, right?

Reply:
correct! it should not interfere. I slowed down the transition to test it:


https://github.com/user-attachments/assets/c72f7391-2e0b-47e1-82dc-4210e9e86d8c

Reply:
Thanks!